### ISSUE TITLE
User "Lycée Louis Pasteur de Bogota" not identified

### ISSUE THREAD

**Issue by angoca** [osmbe/osm-welcome-belgium#440]
Hi, I have configured the Welcome tool for Bogota, and for Colombia, and in parallel I have the Pascal Neis tool to report new users by RSS. However, the user "Lycée Louis Pasteur de Bogota" has not been identified in the Bogota region, and it has already done many changes.

The RSS is http://resultmaps.neis-one.org/newestosmfeed.php?lon=-74.109993&lat=4.6487085&deg=0.38 and the user started on March 7th.

This is the list of users from Bogota https://welcome.osm.be/bogota/list

What could be causing this difference? thank you

**Comment by angoca** [comment]
My ticket seems similar to [399](https://github.com/osmbe/osm-welcome-tool/issues/399)

**Comment by jbelien** [comment]
Probably related indeed.
I fully rely on OSMCha API for mappers detection and ResultMap probably uses a different way of detecting new mappers.
I'll need to investigate further.

**Comment by jbelien** [comment]
This issue is now most likely fixed (see #801) !
There was an issue with the daily update if one of the new mapper detected by OSMCha deleted his/her account meanwhile.
There was another issue if there were too many new mappers ; the request to get their details was truncated.
